src/pkg/config/config.go

```diff
@@ -38,6 +38,8 @@ const InteractiveModeKey = "interactive"
 const DebugKey = "debug"
 const DebugDefault = false
 const OutputKey = "output"
+const OutputPathKey = "output-path"
```

Comment:
This is only used by visualize, so put the flag there instead of in the global config.